myapp.rb

```diff
@@ -52,6 +52,7 @@
   enable :sessions
   enable :logging
   ActiveRecord::Base.logger = Logger.new($stdout)
+  set :public_folder, "#{__dir__}/static"
```

Comment:
What does this do?

Reply:
We already do this for the other two configurations. It essentially just points the app towards its respective styling. We didn't need styling for tests before, but with e2e tests, it's relevant to have the same styling as we do in production :)